sonic-net/sonic-buildimage
# Add Wistron SW-TO3200K and 6512-32R platform support

Signed-off-by: RogerX87 <RogerX87@gmail.com>


Add sw-to3200k and 6512-32r support for 202111

#### Why I did it
Add sw-to3200k and 6512-32r support for 202111

#### How I did it
Add code in device and platform folder for 6512-32r
Update sw-to3200k code both in device and platform folder

#### How to verify it
Install on Wistron device and run command to verify, please check attachment for log
Signed-off-by: RogerX87 <RogerX87@gmail.com>

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)

[3200k_202111.log](https://github.com/sonic-net/sonic-buildimage/files/9400569/3200k_202111.log)
